manifests/g/GeoGebra/GeoGebra5/5.0.672.0/GeoGebra.GeoGebra5.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: GeoGebra.GeoGebra.5

```

```suggestion
PackageIdentifier: GeoGebra.GeoGebraClassic.5
```
Oops, realized that there's a folder `GeoGebraClassic` in `manifests\g\GeoGebra`

Please make sure to move the package to `manifests\g\GeoGebra\GeoGebraClassic\5\5.0.672.0`